src/components/AddLiquidity.tsx

```diff
@@ -145,6 +145,7 @@ function AddLiquidity() {
         )
         console.log(`add liquidity succeed, tx id: ${result.txId}`)
         setTxId(result.txId)
+        updateBalanceForTx(result.txId)
```

Comment:
it seems we did not add this as dependency?

Reply:
Fixed, but I just tested it and there is no problem even if it is not added to the dependency list.